src/lv_api_map_v9_1.h

```diff
@@ -27,6 +27,11 @@ extern "C" {
  * GLOBAL PROTOTYPES
  **********************/
 
+static inline void lv_calendar_set_showed_date(lv_obj_t * obj, uint32_t year, uint32_t month)
```

Comment:
Can it be a `define` similar to the others below?

Reply:
> Can it be a `define` similar to the others below?

Done!  😄